README.md

```diff
@@ -23,7 +23,7 @@ For example with lazy.nvim:
       -- segments = {
       --   { text = { builtin.foldfunc }, click = "v:lua.ScFa" },
       --   {
-      --     sign = { name = { "Diagnostic" }, maxwidth = 2, auto = true },
+      --     sign = { namespace = { "diagnostic/signs" }, maxwidth = 2, auto = true },
```

Comment:
The namespace name is defined here: https://github.com/neovim/neovim/pull/26568/files#diff-212294e88097d8dc45a881c6c3c5fd4ed266040e49fb07317005969849a70156R881

Tested locally on neovim 0.10, and this works for me!